### PR TITLE
Consistent formatting and style

### PR DESCRIPTION
**Pull request by lvignoli**
Fixing #319

### CODE REVIEW

File: .flake8 [hunk 1]
```diff
@@ -1,15 +1,17 @@
...
-exclude = ./build, ./docs
 docstring-convention = google
+exclude = ./build, ./docs
+extend-ignore =
+    # D105 Missing docstring in magic method
+    D105,
+    # E203 whitespace before ':' (for compliance with black)
+    E203,
+# for compliance with black
+max-line-length = 88
```
Comment (HGSilveri) [inline .flake8:10]: When we adopted `black`, we decided to stick with the `flake8` 80 line limit instead

**Review by HGSilveri — COMMENTED** [review]
LGTM in general, just note the detail about the line length

File: .flake8 (hunk 1 above)
Comment (lvignoli) [reply .flake8]: Indeed, it's fixed!

File: .flake8 [hunk 2]
```diff
@@ -7,7 +7,7 @@ extend-ignore =
     # E203 whitespace before ':' (for compliance with black)
     E203,
 # for compliance with black
-max-line-length = 88
+max-line-length = 80
```
Comment (HGSilveri) [inline .flake8:10]: Eish, I'm sorry, I just checked and it's actually 79... Maybe just remove this line instead, since we're using the `flake8` default

**Review by HGSilveri — APPROVED** [review]

**Pull request merged**